tableauserverclient/server/endpoint/schedules_endpoint.py

```diff
@@ -71,10 +74,12 @@ def add_to(resource, type_, req_factory):
             id_ = resource.id
             url = "{0}/{1}/{2}s".format(self.siteurl, schedule_id, type_)
             add_req = req_factory(id_)
-            _ = self.put_request(url, add_req)
-            # TODO: Assert that server_response is 2xx, otherwise, throw an error
+            response = self.put_request(url, add_req)
+            if response.status_code < 200 or response.status_code >= 300:
+                return AddResponse(result=False,
+                                   error="Status {}: {}".format(response.status_code, response.reason))
             logger.info("Added {} to {} to schedule {}".format(type_, id_, schedule_id))
-            return True
+            return OK
```

Comment:
The other `PUT` endoints just return `True` or `None` or something equally meaningless; looks like you want to touch that concept up?

I think the idea of an `OK` is solid -- it won't bother anyone who doesn't care, and it's a little cleaner for those who do. I'm mulling over the real usage of it.. would it be something like:

```
from blah import OK
resp = server.schedules.add_to_schedule(schedule_id, workbook)
if resp is not OK:
    error_stuff()
# OR

resp = server.schedules.add_to_schedule(schedule_id, workbook)
if not resp.is_ok():
    error_stuff()

# OR

resp = server.schedules.add_to_schedule(schedule_id, workbook)
if not resp:  # I don't know how truthy-ness of namedtuples works but we can probably overwrite __bool__ 
    error_stuff()
```

Or something else?

I'm not a big fan of requiring users to import `OK` if they want to check that a response went well, but I agree the OK concept is better than just raw 'True'